website/docs/basics/blocks-special-use-cases.md

```diff
@@ -6,5 +6,84 @@ sidebar_label: Special Use Cases
 
 [![docs-source](https://img.shields.io/badge/source-eigthshift--frontend--libs-yellow?style=for-the-badge&logo=javascript&labelColor=2a2a2a)](https://github.com/infinum/eightshift-frontend-libs/tree/develop/blocks/init/src/blocks/)
 
+By special use cases, we mean just that **special**.
```

Comment:
This sentence is a bit unclear. I'd remove it.

Reply:
I will just remove it